doc/rbd/iscsi-target-cli-manual-install.rst

```diff
@@ -0,0 +1,148 @@
+==================================
+Manual ceph-iscsi-cli Installation
+==================================
+
+**Requirements**
+
+The following packages will be used by ceph-iscsi-cli and target tools.
+They must be installed from your Linux distribution's software repository
+on each machine that will be a iSCSI gateway:
+
+-  libnl3
+-  libkmod
+-  librbd1
+-  pyparsing
+-  python kmod
+-  python pyudev
+-  python gobject
+-  python kmod
+-  python pyudev
+-  python gobject
+-  python urwid
+-  python pyparsing
+-  python rados
+-  python rbd
+-  python netaddr
+-  python netifaces
+-  python crypto
+-  python requests
+-  python flask
+-  pyOpenSSL
+
+**Packages**
+
+-  Linux Kernel
+
+   If not using a distro kernel that contains the required Ceph iSCSI patches,
+   then Linux kernel v4.17 or newer or the ceph-client ceph-iscsi-stable
+   branch must be used. To get the ceph-iscsi-stable branch run:
+
+   ::
+
+       > git clone https://github.com/ceph/ceph-client.git
+       > git checkout ceph-iscsi-stable
+
+   Check your distro's docs for specific instructions on how to build a
+   kernel. The only Ceph iSCSI specific requirements are the following
+   build options must be enabled:
+
+   ::
+
+       CONFIG_TARGET_CORE=m
+       CONFIG_TCM_USER2=m
+       CONFIG_ISCSI_TARGET=m
+
+-  tcmu-runner
+
+   Installation:
+
+   ::
+
+       > git clone https://github.com/open-iscsi/tcmu-runner
+       > cd tcmu-runner
+
+   Ceph iSCSI requires systemd so the following build command must be used:
+
+   ::
+
+       > cmake -Dwith-glfs=false -Dwith-qcow=false -DSUPPORT_SYSTEMD=ON -DCMAKE_INSTALL_PREFIX=/usr
+       > make install
+
+   Enable and start the daemon:
+
+   ::
+
+       > systemctl enable tcmu-runner
```

Comment:
Nit: does ```make install``` reload systemd to pick up the new script?

Reply:
It does not. Fixed.